eks-distro-base/check_update.sh

```diff
@@ -18,15 +18,12 @@ set -e
 set -o pipefail
 set -x
 
-AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-AWS_REGION="us-west-2"
-BASE_IMAGE=${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/eks-distro/base:$(cat eks-distro-base/TAG_FILE)
-mkdir eks-distro-base/check-update
-cat << EOF >> eks-distro-base/check-update/Dockerfile
+BASE_IMAGE=${1}/${2}:$(cat TAG_FILE)
```

Comment:
Just so the args are clear, let's define them at the top and reference them here so we have a better idea of what those values are

Reply:
Done